test/yubico_test.sh

```diff
@@ -0,0 +1,18 @@
+# Requires dfu-utils to be installed.
+# This runs through the steps described in https://github.com/Yubico/libu2f-host documentation
+set -x
+curl 'https://demo.yubico.com/wsapi/u2f/enroll?username=tomu&password=test' > regchallenge.json
+cat regchallenge.json
+echo "Touch key to confirm registration"
+u2f-host -a register -o 'https://demo.yubico.com'  < regchallenge.json > regdata.json
```

Comment:
Shall we add .json files to .gitignore and clean them up at the end of this script?

Reply:
OK

Reply:
Now by default they are cleaned up at end, unless you add "-k" as first argument.  Rather simplistic, but I think it is ok for a basic test script.